core/update_system.py

```diff
@@ -476,6 +479,7 @@ def process_from_nodes(nodes):
             print("Something not very important happend in Blender memory", node, type(node))
 
     ng = nodes[0].id_data
+    check_fake_user(ng)
```

Comment:
```suggestion
    ng.use_fake_user = True
```

Reply:
Are you sure that would be something to do on every update? thats why a i added the check, I dont know if something else happens inside blender when use_fake_user is set

Reply:
I think it is static property which is taking into account when Blender makes decisions of clearing data.

Reply:
Probably even ban could be done. We can override creating new tree operator.
![2020-10-15_12-38-59](https://user-images.githubusercontent.com/28003269/96098691-76e4c000-0ee3-11eb-9d71-37d7d705a00b.png)
Or we can try to subscribe to `node_tree` attribute changes of a windows manager.

Reply:
Yes subscribesion works
```Python
bpy.msgbus.subscribe_rna(
        key=(bpy.types.SpaceNodeEditor, 'node_tree'),
        owner=object(),
        args=(),
        notify=lambda *args: print(f'Tree was changed, probably new one was created')
    )
```

Reply:
@Durman Should I close this PR and let you do it in a more efficient way?